src/test/argsman_tests.cpp

```diff
@@ -0,0 +1,1043 @@
+// Copyright (c) 2011-2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <util/system.h>
+#include <fs.h>
+#include <sync.h>
+#include <test/util/logging.h>
+#include <test/util/setup_common.h>
+#include <test/util/str.h>
+#include <util/strencodings.h>
+#include <univalue.h>
+
+#include <array>
+#include <optional>
+#include <cstdint>
+#include <cstring>
```

Comment:
**nit:** These dependencies can be removed

```suggestion
```

Reply:
No, they are needed, see https://en.cppreference.com/w/cpp/types/integer

Reply:
Okay, I verified that two dependencies are indeed needed for the file.

Reply:
Thanks a lot for the name. I was searching for this tool, but I forgot its name.
I will remember to use it from now on. 😄